api/src/test/java/uk/gov/dhsc/htbhf/dwp/DWPIntegrationTests.java

```diff
@@ -173,7 +179,21 @@ private void assertResponseCorrect(ResponseEntity<EligibilityResponse> response,
         assertThat(response.getStatusCode()).isEqualTo(OK);
         EligibilityResponse eligibilityResponse = response.getBody();
         assertThat(eligibilityResponse).isNotNull();
-        assertThat(eligibilityResponse).isEqualTo(expectedResponse);
+        assertEligibilityResponse(expectedResponse, eligibilityResponse);
+    }
+
+    // checks that two eligibility responses are equal whilst ignoring the order of the children
+    private void assertEligibilityResponse(EligibilityResponse expectedResponse, EligibilityResponse eligibilityResponse) {
+        assertThat(eligibilityResponse).isEqualToIgnoringGivenFields(expectedResponse, "children");
+        assertChildren(eligibilityResponse.getChildren(), expectedResponse.getChildren());
+    }
+
+    private void assertChildren(List<ChildDTO> expected, List<ChildDTO> actual) {
+        if (expected == null) {
```

Comment:
As per other PR, do we need the null check when we know its not null in the test its used in?

Reply:
It's nice to keep these assertion methods generic, especially if we add additional tests in the future.